ethereumj-core/src/main/java/org/ethereum/vm/DataWord.java

```diff
@@ -32,30 +32,31 @@ public DataWord() {
     }
 
     public DataWord(int num) {
```

Comment:
@ligi : that is wrong , the DataWord by 
definition is 32 bytes even if all are zeroes 
fix it in all other places as well